examples/web/index.html

```diff
@@ -501,6 +501,175 @@ <h2 class="typography-heading-medium-text">Alert</h2>
+                      <a href="#" class="Header__navLink">Zaměstnavatelé</a>
+                    </li>
+                  </ul>
+                </div>
+                <div class="Header__actions Header__actions--secondary">
+                  <a href="#" class="Button Button--primary">Přihlásit</a>
+                  <a href="#" class="Button Button--inverted">Vstup pro firmy</a>
+                </div>
+              </div>
+            </nav>
+          </header>
+
+          <script>
+            const HEADER_TOGGLE_SELECTOR = '[data-toggle="header"]';
+            const HEADER_BREAKPOINT = 1280;
+            const OPEN_CLASSNAME = 'is-open';
+
+            window.addEventListener('DOMContentLoaded', () => {
+              const getHeaderSelectorsByToggles = (toggleSelector) => {
+                const headerSelectors = [];
+
+                document.querySelectorAll(toggleSelector).forEach((toggleEl) => {
+                  const target = toggleEl.dataset.target;
+
+                  if (!headerSelectors.includes(target)) {
+                    headerSelectors.push(target);
+                  }
+                });
+
+                return headerSelectors;
+              };
+
+              const toggleHeader = (targetEl) => {
+                // Only toggles inside current header are handled.
+                const targetToggleEls = targetEl.parentElement.querySelectorAll(HEADER_TOGGLE_SELECTOR);
+
+                if (targetEl.classList.contains(OPEN_CLASSNAME)) {
+                  targetEl.classList.remove(OPEN_CLASSNAME);
+                  targetToggleEls.forEach((toggleEl) => toggleEl.setAttribute('aria-expanded', 'false'));
+                  // window.removeEventListener('click', handleClickOutside);
+                  document.removeEventListener('keydown', handleKeyPress);
+                  window.removeEventListener('resize', handleWindowResize);
+                  console.log('close');
+                } else {
+                  targetEl.classList.add(OPEN_CLASSNAME);
+                  targetToggleEls.forEach((toggleEl) => toggleEl.setAttribute('aria-expanded', 'true'));
+                  // window.addEventListener('click', handleClickOutside);
+                  document.addEventListener('keydown', handleKeyPress);
+                  window.addEventListener('resize', handleWindowResize);
+                  console.log('open');
+                }
+              };
+
+              const closeOpenHeaders = () => {
+                const headerSelectors = getHeaderSelectorsByToggles(HEADER_TOGGLE_SELECTOR);
+
+                if (headerSelectors.length) {
+                  document.querySelectorAll(headerSelectors.join(', ')).forEach((headerEl) => {
+                    if (headerEl.classList.contains(OPEN_CLASSNAME)) {
+                      toggleHeader(headerEl);
+                    }
+                  });
+                }
+              };
+
+              const handleHeaderToggleClick = (e) => {
```

Comment:
Please use the full name instead of the abbreviation. I know that `e` common abbreviation for an element but it improves readibility.
```suggestion
              const handleHeaderToggleClick = (element) => {
```

Reply:
Good point because `e` stands for `event` 🙂

Reply:
🤦 Reading `headerEl`, `toggleEl` and now `e`, another `element` :D